packages/flame/lib/src/particles/circle_particle.dart

```diff
@@ -2,20 +2,19 @@ import 'dart:ui';
 
 import 'particle.dart';
 
-/// Plain circle with no other behaviors
-/// Consider composing with other [Particle]
-/// to achieve needed effects
+/// Plain circle with no other behaviors.
+///
+/// Consider composing with other [Particle] to achieve needed effects.
 class CircleParticle extends Particle {
   final Paint paint;
+
```

Comment:
I think you can add these empty lines when you add the dartdocs instead? So that there aren't empty lines forgotten in the upcoming PR.
It doesn't follow the current pattern we have.